test/utils/image/manifest.go

```diff
@@ -207,7 +206,7 @@ const (
 
 func initImageConfigs() map[int]Config {
 	configs := map[int]Config{}
-	configs[Agnhost] = Config{e2eRegistry, "agnhost", "2.8"}
+	configs[Agnhost] = Config{promoterE2eRegistry, "agnhost", "2.9"}
```

Comment:
Why not 2.10?

Reply:
2.10 is not yet promoted: https://github.com/kubernetes/k8s.io/pull/562

I have suggested that we add more conformance-related people to the OWNERS file which could potentially help promote the images.

Reply:
Oh, I see, the other 2.10 references are in README files. I have pinged Linus (@listx), hopefully he can merge that soon.